plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -1215,7 +1223,7 @@ def update_needed(self, other):
                             format.
         """
         if other.updated == "":
-            return False
+            return other.version > self.version
```

Comment:
hmm I had assumed `version` was required since it was neither marked as optional, nor commented on here: https://github.com/openSUSE/libzypp/blob/master/zypp/parser/yum/schema/updateinfo.rnc#L11

But I guess I shouldn't assume anything when it comes to errata...